internal/commands/profile/init.go

```diff
@@ -49,6 +56,16 @@ func NewCmdInit(ctx *cmd.Context) *cmd.Command {
 
 			return opts.run()
 		},
+		Flags: cmd.Flags{
+			Local: []*cmd.Flag{
+				{
+					Name:          "vault-secrets",
+					Description:   "Flag to configure current active profile for use with HCP Vault Secrets.",
```

Comment:
Description shouldn't include that it is a flag. I also think we can drop HCP since it is the `hcp` cli. 

```suggestion
					Description:   "Initialize Vault Secrets configuration.",
```